internal/flags.go

```diff
@@ -43,6 +43,8 @@ const (
 	flagLogFileSize     = "logFileSize"
 	flagLogFileCount    = "logFileCount"
 	flagLogFileMaxAge   = "logFileMaxAge"
+	flagRetryCount      = "downloadRetryCount"
```

Comment:
Could you please create a new ticket for the documentation?

Reply:
https://github.com/eclipse-kanto/kanto/issues/96